src/main/java/core/basesyntax/Shape.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public abstract class Shape implements Figure {
+    private String color;
+
+    public Shape(String color) {
+        this.color = color;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public void setColor(String color) {
+        this.color = color;
+    }
+
+    public abstract double getArea();
```

Comment:
Don’t use abstract classes to set behaviour for classes. Also, don’t put all behaviour into a single interface if the methods are conceptually different from each other. 

So, let's create another interface `AreaCalculator`, where you will define the method `getArea()`. Don't forget to add this new interface to `implements` statement in this class.